inference-engine/include/ie_blob.h

```diff
@@ -577,13 +577,14 @@ class TBlob : public MemoryBlob {
     /**
      *@brief Virtual destructor.
      */
-#ifdef __clang__
+
+#if defined(__clang__) && !defined(__SYCL_COMPILER_VERSION)
     virtual ~TBlob();
 #else
     virtual ~TBlob() {
         free();
     }
-#endif  // __clang__
+#endif  // __clang__ && !__SYCL_COMPILER_VERSION
```

Comment:
Missed change in https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/inference_engine/ie_rtti.cpp